packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/Grounding.java

```diff
@@ -23,12 +26,14 @@ public class Grounding implements Serializable {
 	private static final long serialVersionUID = 302308407252037615L;
 
 	/** Ontological identifier per DKG */
-	@JdbcTypeCode(SqlTypes.JSON)
+	@Type(JsonType.class)
+	@Column(columnDefinition = "text")
```

Comment:
These will require a migration

Reply:
(and the other type changes will too - I won't comment spam on them :)

Reply:
No migration needed, as we reverted datasets and these Groundings will all be created fresh in their own table.